doc/maintainers/team_process.md

```diff
@@ -178,8 +178,72 @@ maintainers team makes no guarantees about regularly checking them.
 now the most important thing is to direct users to a single destination.*
 
 
-## Old issue closing policy
+## Issue closing policy
 
-We have yet to settle on a precise guide for closing out old issues, but, for
-now, if you happen across something that could easily be marked as resolved,
-please do so.
+We need to strike a balance in issue handling between keeping a maintainable
+set of issues&mdash;thereby making the lives of maintainers, contributors, and
+users easier&mdash;and allowing legitimate issues to remain open.
+
+The policy for closing an issue depends entirely on the type of issue we're
+looking at.
+
+When closing an issue, please provide a brief explanation for why the issue was
+closed, such as a reference to a PR, a comment about lack of clarification,
+etc.
+
+### Discussion
+
+Some issues are for discussing the project. These issues should start with a
+clear question, have an owner, and be closed when either:
+
+* The original question has been answered
+* There is no clear question or owner
+* No progress has been made in 30 days
+
+Discussions are typically opened by maintainers and contributors.
+
+### Support
+
+Support questions are requests for assistance in using Stack. They must have a
+clear set of steps followed by the user and a desired outcome. Support requests
+sometimes overlap with bug reports, handled next. Supports requests are closed
+when:
+
+* The user decides their question is answered
+* No clarification to questions from the Stack team is given for 7 days
+* The question is determined out of scope by the Stack team
+
+### Bug report
+
+A bug report describes a misbehavior in Stack itself. There are some cases
+where an issue may appear somewhere between a support request and a bug report.
+The Stack team will determine which category an issue falls into. Bug reports
+must include clear reproduction steps, an expected outcome, and an actual
```

Comment:
I know that these are often all necessary to resolve the bug report, but this language is quite strong. Some bugs can't be reproduced reliably (‘clear reproduction steps’) e.g. some race conditions. Even without these details, a version and platform is often all that is needed to diagnose known bugs.

Reply:
What I'm trying to establish here is "what are the rules under which a bug report will be closed by the maintainer team." Too often I see vague bug reports that linger on without any ability to reproduce. I don't think anyone is going to lawyer issues and say "well, I totally understand what you're saying and can reproduce, but you didn't fill out the form correctly." I'm open to rewording though.

Reply:
> what are the rules under which a bug report will be closed by the maintainer team

Yes, that will be helpful, especially for triage. In that case, the wording is OK.

Reply:
Adding this explanation.